integration/solana/debug_buffer.spec.ts

```diff
@@ -0,0 +1,148 @@
+// SPDX-License-Identifier: Apache-2.0
+
+import { Connection, Keypair, PublicKey, sendAndConfirmTransaction, SystemProgram, Transaction } from '@solana/web3.js';
+import expect from 'expect';
+import { loadContract } from './setup';
+import { Program, Provider, BN } from '@project-serum/anchor';
+import path = require("path")
+
+describe('DebugBuffer', function () {
+    this.timeout(150000);
+
+    let program: Program;
+    let storage: Keypair;
+    let payer: Keypair;
+    let provider: Provider;
+
+    before(async function () {
+        ({ program, storage, payer, provider } = await loadContract('Printer'));
+    });
+
+    it('Prints runtime errors', async function () {
+        let absolutePath = path.resolve("debug_buffer.sol");
```

Comment:
I'm not sure we need the whole path. Maybe just the file name. What do you think @xermicus?

We need reproducable builds. If the path depends on this then that's a pain

Reply:
Good point, I had the same thought (alas the full path is not necessary), but reproducible builds didn't came to my mind. But with that in mind, I think it would better to just use only the file name. Otherwise we have a CLI flag that breaks reproducible builds, which is something we should be keen to avoid if possible.